### ISSUE TITLE
LDAP Integration

### ISSUE THREAD

**Issue by jask87** [sapplica/sentrifugo#154]
Is there a plan to integrate LDAP? I would be willing to help/do the development.

**Comment by sapplica** [comment]
Hi,

Thank you for showing interest in our product! We are sorry to disappoint you. As of now, we don't have any plans to implement LDAP authentication.

**Comment by tarassh** [comment]
Hello,

I have implemented LDAP auth.

**Comment by UlianaKrainska** [comment]
Dear Sertifugo Team,
As Taras mentioned in previous comment Eleks.com has implemented LDAP auth and we're ready to contribute.

It'll be great to speak with Product Manager and discuss this opportunity.

BR,

**Comment by jask87** [comment]
How can i implement that @tarassh ?

**Comment by tarassh** [comment]
I am preparing the code for a pull request.

**Comment by jask87** [comment]
Sweet let me know when its ready.

**Comment by martimarkov** [comment]
@sapplica Will you be accepting this PR? It's been open since the 2 Aug and it's definitely the my only blocker for using Setrifugo.

**Comment by compumatter** [comment]
I second that.  Would really like to use this software but LDAP needed

**Comment by atelierscym** [comment]
Hello,
If of any help, I wish as well have an import/syncro option towards ldap.
Thanks. I'm available to participate in writing use cases and requirements.

**Comment by FlorianHeigl** [comment]
is there a reason why all integration related tickets like api or ldap backends remain unanswered for years? do they somehow conflict with project goals?

**Comment by mssystem** [comment]
Yes an ldap/active directory feature would be a much appreciated!

**Comment by noblemfd** [comment]
@tarassh  Can you please share how you implemented LDAP/Active Directory with Sentrifugo

**Comment by noblemfd** [comment]
@tarassh Can you please share how you implemented LDAP/Active Directory with Sentrifugo  my email is noblemfd@gmail.com

**Comment by tanist-johnlewis** [comment]
@tarassh are you still alive? Did you ever push?

**Comment by ceekaymats** [comment]
@tarassh Did you manage to push the LDAP/Active Directory with Setrifugo?? Could really use your help
